system/atomic_image.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION='''
+---
+module: atomic_image
+short_description: Manage the container images on the atomic host platform
+description:
+    - Manage the container images on the atomic host platform
+version_added: "2.2"
+author: "Saravanan KR @krsacme"
+notes:
+    - Host should be an atomic platform (verified by existence of '/run/ostree-booted' file)
+requirements:
+  - atomic
+options:
+    name:
+        description:
+          - Name of the container image
+        required: True
+        default: null
+    state:
+        description:
+          - Whether to install (C(present) or C(start) or C(update)), or uninstall (C(stop) or C(remove))
+        required: True
+        choices: ["present", "start", "stop", "update", "remove"]
+        default: null
+'''
+
+EXAMPLES = '''
+
+# Install rhel-tools container image (atomic install rhel7/rhel-tools)
+- atomic_image: name=rhel7/rhel-tools state=present
+
+# Run rsyslog container image (atomic run rhel7/rsyslog)
+- atomic_image: name=rhel7/rsyslog state=start
+
+'''
+
+RETURN = '''
+msg:
+    description: The command standard output
+    returned: always
+    type: string
+    sample: [u'Using default tag: latest ...']
+'''
+
+import os
+import sys
+
+def core(module):
+    state_map = {'present'  : 'install',
+                 'start'    : 'run',
+                 'stop'     : 'stop',
+                 'update'   : 'update',
+                 'remove'   : 'uninstall'}
```

Comment:
Agreed. With changing `state` to `command` these mappings do seem unnecessary and confusing and should be removed.